.gitignore

```diff
@@ -39,4 +39,8 @@ Tests/TestVars
 .Spotlight-V100
 .Trashes
 ehthumbs.db
-Thumbs.db
\ No newline at end of file
+Thumbs.db
+
+# Custom Test Config #
+######################
+Vester/Configs/config.json
```

Comment:
Yes, the config.json is needed when you run tests. But why it should be in the git-repo? Here is only user specific data in the config.